BridgeAppSDK/SBAProfileManager.swift

```diff
@@ -113,13 +113,20 @@ open class SBAProfileManager: SBADataObject, SBAProfileManagerProtocol {
      */
     public static let shared: SBAProfileManagerProtocol? = {
         SBABridgeManager.addResourceBundleIfNeeded()
-        guard let json = SBAResourceFinder.shared.json(forResource: SBAProfileItemsJSONFilename),
-                let sharedProfileManager = SBAClassTypeMap.shared.object(with:json, classType:SBAProfileManagerClassType) as? SBAProfileManagerProtocol
-            else {
-                // The SBAUser will default to the profile manager if available, but if not will fall-back
-                // to older methods for storing information.
-                return SBAProfileManager(dictionaryRepresentation: [:])
+        let bundles = SBAInfoManager.shared.resourceBundles.reversed()
```

Comment:
I am surprised that this doesn't put you into an infinite loop of wacky madness. The values in `resourceBundles` are changed during set up of the bridge connection to include BridgeAppSDK. Something to watch out for if it doesn't seem to be behaving as expected.